lib/utils.coffee

```diff
@@ -18,7 +18,6 @@ Promise = require('bluebird')
 { fetch: normalFetch, Headers: HeadersPonyfill } = require('fetch-ponyfill')({ Promise })
 urlLib = require('url')
 qs = require('qs')
-parseInt = require('lodash/parseInt')
```

Comment:
I guess we don't support clients that don't use `10` as the radix

Reply:
All the usage specifies `10` already so it doesn't matter what they use as a default :P

Reply:
I didn't look at the code :)